Controls/DataControls/DataControls.UWP/ListView/View/RadListView.Reorder.cs

```diff
@@ -68,21 +68,33 @@ void IReorderItemsHost.CommitReorderOperation(int sourceIndex, int destinationIn
         {
         }
 
-        internal void PrepareReorderItem(RadListViewItem reorderItem)
+        internal void PrepareReorderItem(IReorderItem reorderItem)
```

Comment:
As the reorderItem could be either RadListViewItem or ListViewGroupHeader (both are dependency objects) we can keep the current implementation and simply change the parameter to DependencyObject: `internal void PrepareReorderItem(DependencyObject reorderItem)
        {
            if (reorderItem != null)
            {
                ((IReorderItem)reorderItem).Coordinator = this.reorderCoordinator;
                DragDrop.SetAllowDrop(reorderItem, true);
            }
        }`